main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	pb "github.com/kmala/example-grpc-server/_proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+// main starts an HTTP server listening on $PORT which dispatches to request handlers.
+func main() {
+	http.Handle("/healthz", http.HandlerFunc(healthcheckHandler))
+	// wrap the poweredByHandler with logging middleware
+	http.Handle("/", logRequestMiddleware(http.HandlerFunc(poweredByHandler)))
+	port := getenv("PORT", "5000")
+	log.Printf("listening on %v...\n", port)
+	err := http.ListenAndServe(":"+port, nil)
+	if err != nil {
+		panic(err)
+	}
+}
+
+// poweredByHandler writes a response by querying the grpc server.
+func poweredByHandler(w http.ResponseWriter, r *http.Request) {
+	powered := getenv("POWERED_BY", "Deis")
+	server := getenv("SERVER_NAME", "")
+	if server == "" {
+		log.Println("Server not configured")
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte{})
+		return
+	}
+	address := fmt.Sprintf("%s.%s.svc.cluster.local:80", server, server)
```

Comment:
is it completely necessary to use the full DNS name along with port 80? I thought `name.name` could be used in most cases. At the very least the `:80` is a bit overkill for this case :)